### PR TITLE
SB-14581: feat Disabled Api's as per requirement

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

File: platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java [hunk 1]
```diff
@@ -446,7 +446,7 @@
 	 * @param requestMap
 	 * @return
 	 */
-	@RequestMapping(value = "/dialcode/link", method = RequestMethod.POST)
+//	@RequestMapping(value = "/dialcode/link", method = RequestMethod.POST)
```
Comment (amitpriyadarshi) [inline platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java:449]: this is not pushed till now.

File: platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java [hunk 2]
```diff
@@ -545,7 +545,7 @@ protected String getAPIVersion() {
 		return API_VERSION_3;
 	}
 
-	@RequestMapping(value="/retire/{id:.+}", method = RequestMethod.DELETE)
+//	@RequestMapping(value="/retire/{id:.+}", method = RequestMethod.DELETE)
```
Comment (amitpriyadarshi) [inline platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java:548]: uncomment this also.

File: platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java (hunk 1 above)
Comment (rheafernandes) [reply platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java]: resolved

File: platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java (hunk 2 above)
Comment (rheafernandes) [reply platform-modules/manager/src/main/java/org/ekstep/taxonomy/controller/ContentV3Controller.java]: done

**Pull request merged**